rental.rb

```diff
@@ -5,7 +5,7 @@ def initialize(date, book, person)
     @date = date
     @book = book
     @person = person
-    @book.rentals.push(self)
-    @person.rentals.push(self)
+    @book.rentals << self
+    @person.rentals << self
   end
```

Comment:
- [ ] Dear @KUBAHO3, after creating the `rental` successfully, I could not list all the rentals for the person, Please make sure the `List rental` is functioning as expected. This is the most essential part of the project as it will help you learn to create associations clearly.

![image](https://github.com/KUBAHO3/oop-school-library/assets/69077061/74949b80-8186-4cc1-a834-89c6cc83ce7c)

Reply:
Respectively as the project itself suggests we are supposed to enter the id of a person in order to see their rentals. Since we were asked to use random numbers in the first part you had to enter the printed Id alongside the person's age and by that you will be able to view their rentals! so please consider that way

Reply:
@TadesseDev